tutorials/tts/FastPitch_Adapter_Finetuning.ipynb

```diff
@@ -0,0 +1,868 @@
+    "heteronyms_path = os.path.abspath(os.path.join(code_dir, \"scripts\", \"tts_dataset_files\", \"heteronyms-052722\"))\n",
+    "\n",
+    "# Copy and Paste the PITCH_MEAN and PITCH_STD from previous steps (train_manifest) to overide pitch_mean and pitch_std configs below.\n",
+    "PITCH_MEAN=175.48513793945312\n",
+    "PITCH_STD=42.3786735534668"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0d599575",
+   "metadata": {},
+   "source": [
+    "### Important notes\n",
+    "* `+init_from_ptl_ckpt`: initialize with a multi-speaker FastPitch checkpoint\n",
+    "* `~model.speaker_encoder.lookup_module`: remove the pre-trained looked-up speaker embedding\n",
+    "* Other optional arguments based on your preference:\n",
+    "    * batch_size\n",
+    "    * exp_manager\n",
+    "    * trainer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17638cb9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Normally 100 epochs\n",
+    "!cd {code_dir} && python examples/tts/fastpitch_finetune_adapters.py \\\n",
+    "--config-name=fastpitch_align_44100_adapter.yaml \\\n",
+    "+init_from_nemo_model={pretrained_fastpitch_checkpoint} \\\n",
+    "train_dataset={train_manifest} \\\n",
+    "validation_datasets={valid_manifest} \\\n",
+    "sup_data_types=\"['align_prior_matrix', 'pitch', 'speaker_id', 'reference_audio']\" \\\n",
+    "sup_data_path={supp_dir} \\\n",
+    "pitch_mean={PITCH_MEAN} \\\n",
+    "pitch_std={PITCH_STD} \\\n",
+    "~model.speaker_encoder.lookup_module \\\n",
+    "model.train_ds.dataloader_params.batch_size=8 \\\n",
+    "model.validation_ds.dataloader_params.batch_size=8 \\\n",
+    "model.optim.name=adam \\\n",
+    "model.optim.lr=2e-4 \\\n",
+    "~model.optim.sched \\\n",
+    "exp_manager.exp_dir={logs_dir} \\\n",
+    "+exp_manager.create_wandb_logger=True \\\n",
+    "+exp_manager.wandb_logger_kwargs.name=\"tutorial-FastPitch-finetune-adaptation\" \\\n",
+    "+exp_manager.wandb_logger_kwargs.project=\"NeMo\" \\\n",
+    "+exp_manager.checkpoint_callback_params.save_top_k=-1 \\\n",
+    "trainer.max_epochs=10 \\\n",
+    "trainer.check_val_every_n_epoch=10 \\\n",
+    "trainer.log_every_n_steps=1 \\\n",
+    "trainer.devices=1 \\\n",
+    "trainer.strategy=ddp \\\n",
+    "trainer.precision=32"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "402ae669",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# e.g. NeMoTTS_logs/FastPitch/Y-M-D_H-M-S/checkpoints/adapters.pt\n",
+    "last_checkpoint_dir = sorted(list([i for i in (Path(logs_dir) / \"FastPitch\").iterdir() if i.is_dir()]))[-1] / \"checkpoints\"\n",
+    "finetuned_adapter_checkpoint = list(last_checkpoint_dir.glob('adapters.pt'))[0]\n",
+    "print(finetuned_adapter_checkpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33dcbc8f",
+   "metadata": {},
+   "source": [
+    "# 4. Fine-tune HiFiGAN on adaptation data"
```

Comment:
Since this section has so little text/explanation, maybe we can reference/link to the main finetuning tutorial.

https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/FastPitch_Finetuning.ipynb

I do not see any way to link to the specific section though in github, other than to say it is in the "Finetuning HiFi-GAN" section.

Reply:
I add explanation why we don't use existing script to generate mel-spectrograms.